app/components/agenda/agendaitem/agendaitem-controls/component.js

```diff
@@ -60,7 +57,7 @@ export default Component.extend({
   deleteWarningText: computed('agendaitem.{subcase,subcase.agendaitems}', async function () {
     if (await this.isDeletable) {
       return this.intl.t('delete-agendaitem-message');
-    } else if (this.currentSession.isAdmin) {
```

Comment:
Als dit soort dingen automatisch gebeurd zijn, vind ik dit verdomd scary. Dit is functioneel **niet** hetzelfde :fearful:

Reply:
( https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/if...else )

Reply:
Yes, dit is de autofix PR.. we moeten dat inderdaad goed bekijken.. daarom staat hij alvast open

Reply:
Dit lijkt zichzelf terug rechtgezet te hebben met de merge van development enkele dagen terug..
```javascript
deleteWarningText: computed('agendaitem.{subcase,subcase.agendaitems}', async function () {
    if (await this.isDeletable) {
      return this.intl.t('delete-agendaitem-message');
    } if (this.currentSession.isAdmin) {
      return this.intl.t('delete-agendaitem-from-meeting-message');
    }
    return null;
  }),
```